src/components/Wallet/WalletSign.jsx

```diff
@@ -119,29 +77,6 @@ class WalletSign extends React.Component {
     );
   };
 
-  transactionFinalized = () => {
-    const { transaction, changeSlice, updateChangeNode } = this.props;
-
-    const { txid } = transaction;
-    const { spent } = this.state;
-    if (txid !== "" && !spent) {
-      this.setState({ spent: true });
-      const changeAddress = changeSlice.multisig.address;
-      for (let i = 0; i < transaction.outputs.length; i += 1) {
-        if (changeAddress === transaction.outputs[i].address) {
-          updateChangeNode({
```

Comment:
This is what was bumping the `nextNode` in change and causing the calls to not reflect the updated balance. This isn't necessary since this is being done with our new walletAction, which checks the actual state of the change slice (which may not even be being used)